ArcGISRuntimeSDKQt_CppSamples/Routing/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.cpp

```diff
@@ -138,47 +138,45 @@ void FindServiceAreasForMultipleFacilities::connectServiceAreaTaskSignals()
+    m_serviceAreaTask->createDefaultParametersAsync().then(this, [this](ServiceAreaParameters serviceAreaParameters)
+    {
+      // once default parameters created, set parameters and solve
+      serviceAreaParameters.setPolygonDetail(ServiceAreaPolygonDetail::High);
+      serviceAreaParameters.setReturnPolygons(true);
 
-    QList<double> travelTimes = {1.0, 3.0};
+      QList<double> travelTimes = {1.0, 3.0};
 
-    // add service areas of 1 minute and 3 minutes travel time by car
-    serviceAreaParameters.setDefaultImpedanceCutoffs(travelTimes);
+      // add service areas of 1 minute and 3 minutes travel time by car
+      serviceAreaParameters.setDefaultImpedanceCutoffs(travelTimes);
 
-    // create query parameters used to select all facilities from the feature table
-    QueryParameters queryParameters;
-    queryParameters.setWhereClause("1=1");
+      // create query parameters used to select all facilities from the feature table
+      QueryParameters queryParameters;
+      queryParameters.setWhereClause("1=1");
 
-    // add all facilities to the service area parameters
-    serviceAreaParameters.setFacilitiesWithFeatureTable(m_facilitiesTable, queryParameters);
+      // add all facilities to the service area parameters
+      serviceAreaParameters.setFacilitiesWithFeatureTable(m_facilitiesTable, queryParameters);
 
-    m_future = m_serviceAreaTask->solveServiceAreaAsync(serviceAreaParameters);
-    m_future.then(this, [this](const ServiceAreaResult& serviceAreaResult)
-    {
-      emit taskRunningChanged();
-
-      // iterate through the facilities to get the service area polygons
-      for (int i = 0; i < serviceAreaResult.facilities().size(); ++i)
+      m_future = m_serviceAreaTask->solveServiceAreaAsync(serviceAreaParameters);
+      m_future.then(this, [this](const ServiceAreaResult& serviceAreaResult)
       {
-        QList<ServiceAreaPolygon> serviceAreaPolygonList = serviceAreaResult.resultPolygons(i);
-        // create a graphic for each available polygon
-        for (int j = 0; j < serviceAreaPolygonList.size(); ++j)
+        emit taskRunningChanged();
+
+        // iterate through the facilities to get the service area polygons
+        for (int i = 0; i < serviceAreaResult.facilities().size(); ++i)
         {
-          m_serviceAreasOverlay->graphics()->append(new Graphic(serviceAreaPolygonList[j].geometry(), m_fillSymbols[j], this));
+          QList<ServiceAreaPolygon> serviceAreaPolygonList = serviceAreaResult.resultPolygons(i);
+          // create a graphic for each available polygon
+          for (int j = 0; j < serviceAreaPolygonList.size(); ++j)
+          {
+            m_serviceAreasOverlay->graphics()->append(new Graphic(serviceAreaPolygonList[j].geometry(), m_fillSymbols[j], this));
+          }
         }
-      }
+      }).onFailed([](const ErrorException& e)
+      {
+        qWarning() << e.error().message();
+      });
     });
 
-    if (!m_future.isValid())
```

Comment:
Is this even used anymore? Can you remove it?

Reply:
Reverted the change.
`m_futre .onFailed([](const ErrorException& e)` is already logging the warning, so I was wondering do we need both of the warnings.